elide-datastore/elide-datastore-hibernate/src/main/java/com/yahoo/elide/core/hibernate/hql/SubCollectionFetchQueryBuilder.java

```diff
@@ -74,6 +74,7 @@ public Query build() {
             String filterClause = new FilterTranslator().apply(fe, USE_ALIAS);
 
             String joinClause =  getJoinClauseFromFilters(filterExpression.get())
+                    + getJoinClauseFromSort(sorting)
```

Comment:
Logic looks correct - but you are missing two files that need to have their sorting changed.  SubCollectionPageTotalsQueryBuilder and RootCollectionPageTotalsQueryBuilder